### PR TITLE
Nemo 417/correct partner list

### PR DESCRIPTION
**Pull request by mindyschwab**
### Ticket(s)

_Required unless this is just a change to documentation._

(https://airtable.com/appfJZShN8K4tcWHU/pagGSAxi2xYC3VsDu?GOswY=recBjGxRe81xPB9ah)

### Type of change

_Please delete any options that are not relevant_

- [x] Design change

### Description

Footer: removed partner section and updated styling
about page: Updated partner section to airtable iframe

Fixes # _replace with issue number that this fixes_

### Screenshots

Include relevant screenshots.

#### Please include a screenshot of the Figma design file

Screen Shot 2023-11-21 at 9 38 30 PM

Screen Shot 2023-11-21 at 9 37 45 PM

### Checklist:

_Please delete any options that are not relevant_

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation (if applicable)
- [x] My changes generate no new warnings in the console
- [x] There are no new linting errors/problems in my code
- [x] I have filled this PR out fully, and cleaned up any extraneous items so that it is clear and easy to understand

### CODE REVIEW

File: content/about.ts [hunk 1]
```diff
@@ -11,7 +11,7 @@ interface AboutContent {
   body: string[];
   facts: Fact[];
   joinUs: Header;
-  ourPartners: Header;
+  partnersFrame: AirtableEmbed[];
```
Comment (LeoFuna) [inline content/about.ts:14]:
```suggestion
  partnersFrame: AirtableEmbed;
```
`parternsFrame` indicates that is only one frame, i think remove this `[ ]` is more descriptive.
What do you think?

File: pages/about.tsx [hunk 2]
```diff
@@ -166,70 +165,23 @@ export default function AboutPage() {
...
-                {content.ourPartners.subheading2}
-              </Typography>
-            </Grid>
-          </Grid>
-        </PaperSection>
+      <SectionContainer key={content.partnersFrame[0].id} sx={{ margin: 'auto', px: 3, maxWidth: '65rem' }} maxWidth={false}>
+        <Typography variant="h2" sx={{ mt: '3rem' }}>{content.partnersFrame[0].title}</Typography>
+        <Grid container>
+          <iframe
+            id={`${content.partnersFrame[0].id}-partnersFrame`}
```
Comment (LeoFuna) [inline pages/about.tsx:172]:
```suggestion
            id={`${content.partnersFrame.id}-partnersFrame`}
```
by removing `[ ]`, we do not need this `[0]` anymore

Comment (mindyschwab) [reply pages/about.tsx]: implemented

**Review by e-papanicolas — CHANGES_REQUESTED** [review]

File: pages/about.tsx [hunk 3]
```diff
@@ -166,70 +165,23 @@ export default function AboutPage() {
...
-                  mx: 2,
-                }}
-                variant="body2"
-              >
-                {content.ourPartners.subheading2}
-              </Typography>
-            </Grid>
-          </Grid>
-        </PaperSection>
+      <SectionContainer key={content.partnersFrame.id} sx={{ margin: 'auto', px: 3, maxWidth: '65rem' }} maxWidth={false}>
```
[the remark this answers is not in the document]
Comment (mindyschwab) [reply pages/about.tsx]: Good catch! added id

**Review by LeoFuna — APPROVED** [review]
Great Job Mindy! :tada:

File: pages/about.tsx [hunk 4]
```diff
@@ -166,70 +165,23 @@ export default function AboutPage() {
...
-                  mx: 2,
-                }}
-                variant="body2"
-              >
-                {content.ourPartners.subheading2}
-              </Typography>
-            </Grid>
-          </Grid>
-        </PaperSection>
+      <SectionContainer id="our-partners" key={content.partnersFrame.id} sx={{ margin: 'auto', px: 3, maxWidth: '65rem' }} maxWidth={false}>
```
Comment (e-papanicolas) [inline pages/about.tsx:168]: Can we use the variable for the id instead? `content.partnersFrame.id` 🙏🏼

**Review by e-papanicolas — CHANGES_REQUESTED** [review]

File: pages/about.tsx (hunk 4 above)
Comment (mindyschwab) [reply pages/about.tsx]: of course, this is has been updated

**Review by e-papanicolas — APPROVED** [review]

**Pull request merged**